Jenkinsfile

```diff
@@ -924,8 +924,8 @@ pipeline {
                         sconsBuild parallel_build: true,
                                    scons_args: 'PREFIX=/opt/daos TARGET_TYPE=release BUILD_TYPE=debug',
                                    build_deps: 'no'
-                        sh label: 'Fault injection testing using NLT',
-                           script: './utils/docker_nlt.sh --class-name centos7.fault-injection fi'
+                        sh (script:'''./utils/docker_nlt.sh --class-name el8.fault-injection fi''',
+                            label: 'Fault injection testing using NLT')
```

Comment:
We've been following a code style of not using `()` for steps in the `Jenkinsfile` unless necessary (I think only when there are no arguments) and starting `sh` steps with the label argument to help a reader understand the purpose of script that follows.

Reply:
Fixed in next push